docs/source/general/setup.rst

```diff
@@ -128,8 +128,64 @@ In case you are using `openwisp-radius <https://github.com/openwisp/openwisp-rad
     ./manage.py migrate --fake openwisp_radius 0001_initial_freeradius
     ./manage.py migrate
 
-Installing for development
```

Comment:
Actually, I think the sections that remain in the `Contribution.rst` can be moved to `Setup.rst` because they are specific to the setup of freeradius, wouldn't you agree? :smile:

Reply:
I couldn't understand what you meant by this. Can you please explain what you mean by the sections that remain.

Reply:
I mean that the content from `general/contributing.rst` can be moved to `setup.rst` because only repository setup related steps are remaining, right? :smile: